build.sh

```diff
@@ -6,63 +6,77 @@ var argv = require('yargs').argv;
 shell.exec("gulp clean");
 
 var prebidRepoPath = argv.prebidpath || "../Prebid.js/";
+var task = argv.task || "wrapper";
 
 console.log("ARGV ==>", argv);
 
-if(shell.cd(prebidRepoPath).code !== 0) {
-	shell.echo("Couldnt change the dir to Prebid repo");
-	shell.exit(1);
-}
-
 var prebidTaskName = "";
 var openwrapBuildTaskName = "";
 var openwrapWebpackTaskName = "";
-
-
-if ( argv.mode && argv.mode == "test-build") {
-	console.log("Executing test-build");
-	prebidTaskName = "build-bundle-dev --modules=modules.json";
-	openwrapBuildTaskName = "devbundle";
-	openwrapWebpackTaskName = "devpack";
-
-} else if ( argv.mode && argv.mode == "dev-build" ) {
-	console.log("Executing build");
-	prebidTaskName = "build --modules=modules.json";
-	openwrapBuildTaskName = "bundle";
-	openwrapWebpackTaskName = "webpack";
-
-} else if ( argv.mode && argv.mode == "build" ) {
-	console.log("Executing build");
-	prebidTaskName = "bundle --modules=modules.json";
-	openwrapBuildTaskName = "bundle";
-	openwrapWebpackTaskName = "webpack";
-
+var CREATIVE_TASK = "creative"
+
+if (task == CREATIVE_TASK) {
+		console.log("inside creative");
+		if (shell.exec("gulp webpack-creative --mode=" + argv.mode).code !== 0) {
+			shell.echo('Error: webpack task failed');
```

Comment:
mention which task failed, refer below task fail message

Reply:
Code updated to have error specific to creative or wrapper code